lib/pbench/server/api/resources/users_api.py

```diff
@@ -0,0 +1,534 @@
+import jwt
+from flask import request, jsonify, make_response
+from flask_restful import Resource, abort
+from flask_bcrypt import check_password_hash
+from email_validator import EmailNotValidError
+from sqlalchemy.exc import SQLAlchemyError, IntegrityError
+from configparser import NoOptionError
+from pbench.server.database.models.users import User
+from pbench.server.database.models.active_tokens import ActiveTokens
+
+from pbench.server.api.auth import Auth
+
+
+class RegisterUser(Resource):
+    """
+    Abstracted pbench API for registering a new user
+    """
+
+    def __init__(self, config, logger):
+        self.server_config = config
+        self.logger = logger
+        try:
+            self.token_expire_duration = self.server_config.get(
+                "pbench-server", "token_expiration_duration"
+            )
+        except NoOptionError:
+            self.token_expire_duration = None
```

Comment:
yes I added 60 mins default duration

Reply:
Fixed, I updated the value in the server config file to 60 mins

Reply:
An old comrade of mine was found of quoting what he referred to as "Jack's Law" (although, I'm not sure who Jack was...):

> If something is defined in two places, _at least_ one of them will be wrong.

Is it practical for this code to fetch the value from the config file, so that we only define it in one place?

Reply:
I think in that case I can just remove this try catch block since `token_expiration_duration` is embedded in the _default_ config file so we are certain to always find this option

Reply:
So, you're suggesting removing the `try` block but restoring the `token_expire_duration` attribute (initialized from the config file)?  That sounds reasonable to me.

This seems like a value that we need to have, and I think it's a bad idea to have it defined _both_ in the config file and in the code -- so getting it (once) from the config file, assuming/depending on it being there, and dying a horrible death if it's not seems like an appropriate course.